sig-service-catalog/charter.md

```diff
@@ -0,0 +1,131 @@
+# SIG Service Catalog Charter
+Service Catalog is a Kubernetes extension project that implements the [Open Service Broker API](https://www.openservicebrokerapi.org/) (OSBAPI).
+It allows application developers the ability to provision and consume cloud services natively from within Kubernetes.
+
+This charter adheres to the conventions described in the [Kubernetes Charter README](https://github.com/kubernetes/community/blob/master/committee-steering/governance/README.md).
+
+## Scope
+
+### In scope
+
+This SIG’s main goals are:
+- Support, and adhere to, the Platform requirements of the OSBAPI specification.
+- Provide a UX for Kubernetes users that is consistent with both the OSB API specification and traditional Kubernetes user interactions.
+- Align with the [OSBAPI specification](https://github.com/openservicebrokerapi/servicebroker/blob/master/spec.md) as changes are made.
+- Provide feedback (bugs or feature requests) to the OSBAPI WG.
+
+### Out of scope
+
+The following, non-exhaustive, items are out of scope:
+- Operation of OSBAPI Service Brokers.
+
+## Roles
+
+- Maintainers
```

Comment:
Since we don't use OWNERS file yet, we should write down who is Maintainer and who is a Chair, and either have that as part of this doc, or merged into the service-catalog repo beforehand.

Reply:
Seems easy enough to get the OWNERS file cleaned up first?

Reply:
Okay, I think OWNERS is probably fine as is. 

We should say MAINTAINERS == APPROVERS
and
we will move people to being REVIEWERS, and then maintainers/approvers.

Reply:
Chairs are listed in the Sig definition?

Reply:
Yes, the chairs are listed in `sigs.yaml`. I'm not sure I understand your comment about "maintainers and aprpovers and reviewers". Is that something that would happen after we move to PROW?

Otherwise, suggested verbiage would be greatly appreciated! 😀

Reply:
I think we can say for our first draft here, that "Maintainers is equivalent to the standard definition of Approvers".